seatunnel-core/seatunnel-core-base/src/main/java/org/apache/seatunnel/command/FlinkCommandArgs.java

```diff
@@ -20,8 +20,14 @@
 import org.apache.seatunnel.common.config.DeployMode;
 import org.apache.seatunnel.config.EngineType;
 
+import com.beust.jcommander.Parameter;
+
 public class FlinkCommandArgs extends AbstractCommandArgs {
 
+    @Parameter(names = {"--application-mode"},
+        description = "run job in application mode")
+    private boolean isApplicationMode = false;
```

Comment:
I would like to add a new parameter `private String executeMode` rather than add `isApplicationMode`. Or you can add a new enum `FlinkExecuteMode` to represent this.

Reply:
Hi, @ruanwenjun , thanks for you review. Let's have a more detail discuss.

Flink use `run` and `run-application` to do different action.
* run
  * yarn-session, kubernetes-session for `session` mode
  * yarn-per-job for `per-job` mode
* run-application: for application mode
  * kubernetes-application, yarn-application

The sub commands `yarn-session`, `kubernetes-session`, `yarn-per-job`, `kubernetes-application`, `yarn-application` could be config with `execution.target`.

Now I want to distinguish the action 'run' and 'run-application', so I add `--application-mode`. 

In your point, should we use executeMode (value could be 'session', 'per-job', 'application') to do more commands generate with `run`/`run-application` and `execution.target` check? or just `run` and `run-application` judgement?

Thanks.

Reply:
make sense. 

I will update the PR lately.

Thanks.

Reply:
And don't forget to update the doc https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/command/usage.mdx